### PR TITLE
Background color changed

### PR DESCRIPTION
**Pull request by Souvik34**
Custom scroll bar
hover effects on nav  bar

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/components/Header/styles.css
Comment (ShwetKhatri2001) [inline src/components/Header/styles.css:1]: No need to add this long script if we are not using all variables.

**Review by ShwetKhatri2001 — CHANGES_REQUESTED** [review]
First, raise an issue and wait till you get assigned before start working. 

Talking about the changes, the background color needs to be light so that all text elements can be clearly visible. Also, change the orange color being used to some other colour near to new background color you choose.

**Comment by Souvik34** [comment]
I have removed the extra code part of css. It was kept for testing.

**Comment by ShwetKhatri2001** [comment]
Good, have a look at other suggested improvements then we can merge this.

**Comment by Souvik34** [comment]
Updated the color of scroll bar

**Comment by ShwetKhatri2001** [comment]
Can you change with these colours,
Background color : #e3e8ed
Scroll thumb color : #68707a

I can merge this after that.

**Review by ShwetKhatri2001 — APPROVED** [review]
LGTM 🎊 Congrats @Souvik34 on your first PR getting merged on AR-Webstore  🚀 Keep Contributing 👍

**Pull request merged**